SpatialGDK/Source/SpatialGDKEditor/Private/SchemaGenerator/SpatialGDKEditorSchemaGenerator.cpp

```diff
@@ -516,5 +518,10 @@ bool SpatialGDKGenerateSchema()
 
 	AdditionalSchemaGeneratedClasses.Empty();
 
+	if (bHasByRefRPCArgs)
+	{
+		UE_LOG(LogSpatialGDKSchemaGenerator, Warning, TEXT("Pass-by-reference or array arguments detected in some blueprint RPCs. Please check previous log messages for more details."));
+	}
+
```

Comment:
Could you get rid of `bHasByRefRPCArgs` variable and just move the log further up where you do `!ValidateRPCArguments(TypeInfos)` check, as you don't use `bHasByRefRPCArgs` anywhere else?

Reply:
Good point. I'll do that when fixing the serialization issue as this warning will be going away.

Reply:
Oh, ignore me, just read the previous comment and it makes sense to have this at the end of the log output in case there is a lot of it.

Reply:
Oh, and I realized I misread your suggestion. I thought you meant to move the `ValidateRPCArguments` down here.

Reply:
I initially meant moving the warning up, but looks like `ValidateRPCArguments` is only outputting logs and not performing any other logic, so could also move that down.